kubetest/azure.go

```diff
@@ -173,9 +173,6 @@ func newAcsEngine() (*Cluster, error) {
 	if err := os.Setenv("KUBERNETES_CONFORMANCE_TEST", "yes"); err != nil {
 		return nil, err
 	}
-	if err := os.Setenv("KUBERNETES_CONFORMANCE_PROVIDER", "azure"); err != nil {
```

Comment:
side note: I'm really suprised we even have this upstream, cc @spiffxp why does this even exist??

Reply:
https://github.com/kubernetes/kubernetes/blob/master/hack/ginkgo-e2e.sh#L62

Reply:
I was a little bit confused by it, but followed blindly what kops did here: https://github.com/kubernetes/test-infra/blob/12378e13684ab69e1102263e51c1f3e7869fe04e/kubetest/kops.go#L234

Reply:
Good question, no idea, and no time to investigate right now.  Opened https://github.com/kubernetes/kubernetes/issues/71579 to do so

Reply:
thanks!